llvm/test/tools/yaml2obj/ELF/note-section.yaml

```diff
@@ -462,3 +466,32 @@ Sections:
         Desc: 030405
       - Name: GNU
         Type: NT_GNU_BUILD_ID
+
+--- !ELF
```

Comment:
I think you can avoid this almost duplicate YAML blob by using yaml2obj's -D option e.g.
```
yaml2obj -DELFCLASS=32 ...
....

Class: ELFCLASS[[ELFCLASS]]
```